source/MaterialXRender/ShaderValidators/Glsl/GlslProgram.cpp

```diff
@@ -1061,6 +1061,26 @@ const GlslProgram::InputMap& GlslProgram::updateUniformsList()
         // i.e the type indicated by the HwShader does not match what was generated.
         bool uniformTypeMismatchFound = false;
 
+        /// Return constant block of variables for a stage.
```

Comment:
Some code for storage of constant blocks on a GLSL program so can access it and debug.
(Not related to the enum remapping)